docs/src/test/scala/sample/scaladsl/ConsumerExample.scala

```diff
@@ -191,6 +185,57 @@ object AtLeastOnceWithBatchCommitExample extends ConsumerExample {
   def business(key: String, value: Array[Byte]): Future[Done] = ???
 }
 
+// Consume messages at-least-once, and commit in batches
+object PassThroughFlowCommitExample extends ConsumerExample {
+  def main(args: Array[String]): Unit = {
+
+    // #passThroughFlow
+    var writeFlow = Flow[ConsumerRecord[String, Array[Byte]]].map(_ => ???)
+
+    val control =
+      Consumer
+        .committableSource(consumerSettings, Subscriptions.topics("topic1"))
+        .map(msg => (msg.record, msg.committableOffset))
+        .via(PassThroughFlow(
+          writeFlow, // Here you can replace a module
+          Flow[CommittableOffset].map(x => x) // just pass through the offset
```

Comment:
You could just short-circuit the identity flow, or do you anticipate a good use-case for it?

Reply:
Do you mean changing
`unzip.out1 ~> passThroughB ~> zip.in1` to `unzip.out1 ~> zip.in1`? In this case yes, I think that this is the most common scenario and makes the Graph more "semantic". (just tried [here](https://scalafiddle.io/sf/pneGum2/2)).